spec/models/project_spec.rb

```diff
@@ -21,17 +21,18 @@
     expect(sub_project2.position).to eq 2
   end
 
-  describe ".with_ordered_descendents" do
+  describe ".sub_projects_with_ordered_stories" do
```

Comment:
Matching the name with the existing `scope`.
https://github.com/fastruby/points/blob/d0eb2eed881338a83a68a792a68c6c22106eea7c/app/models/project.rb#L16